### ISSUE TITLE
 Audio/Video Call App

### ISSUE THREAD

**Issue by pavoleg2009** [weeeBox/mobile-system-design#15]

**Comment by weeeBox** [comment]
That would be tough to pull-off without relevant experience 🙂

**Comment by pavoleg2009** [comment]
I agree. But may be we may assume that the low-level part is out is of scope and we have some library. One day I was asked to design library to manage microwave ovens :) And it was one of FAANG company.
